src/lib/web-component/log-view/log-view-element.js

```diff
@@ -1,5 +1,7 @@
 import { dateTimeFormat } from '../../formatters.js';
 
+let totalLogs = 0;
```

Comment:
This is not a local variable but a global variable. The value is only used inside `triggerLogCountChange`, so you may as well compute the value from there (`this.tableBody.rows.length` at the moment).